### DISCUSSION TITLE
Potential entry in OSS events for auto-cpufreq

### DISCUSSION THREAD

**Discussion by rootCircle** [AdnanHodzic/auto-cpufreq#570] [General]
Hello @AdnanHodzic! I've been considering the idea of incorporating auto-cpufreq into open-source software events like [Hacktoberfest](https://hacktoberfest.com/) in the future. Given that the project has recently reached version 2.x, it seems like an opportune moment to invite a more extensive and diverse community to contribute.

Of course, such endeavors do come with some commitments. There might be a need to invest additional time in setting up the infrastructure for participation and reviewing incoming pull requests.

I'd like to propose this idea to you, as I believe it could foster greater collaboration and attract more contributors to auto-cpufreq. Your thoughts and insights on this matter would be greatly appreciated!

Thank you!

**Comment by AdnanHodzic** [comment]
Great idea and something that was on my ToDo as well. I registered as "maintainer" and topic `hacktoberfest` has been added to auto-cpufreq repo. So it's game on from my side 🙂

I encourage everyone who wants to participate as part of Hacktoberfest to add `hacktoberfest` label to their PR's. From my side all pull requests that I approve from this point on until end of month I'll add `hacktoberfest-accepted` label.

So please feel free to spread the word, I'll do the same.

**Comment by rootCircle** [comment]
Sure, I will, for the time being, can you kindly post in the project in the [Hacktoberfest Official Discord Server](https://discord.gg/hacktoberfest) to [project-sharing channel](https://discord.com/channels/720788516837261322/735506115810426971).[or if you want, I can post in on your behalf also]

Thanks a lot for accepting in, btw😃

**Comment by AdnanHodzic** [comment]
Of course! Invite to contributors shared on Haktoberfest shared as suggested: https://discord.com/channels/720788516837261322/735506115810426971/1158292438406930442

**Comment by AdnanHodzic** [comment]
@rootCircle please note if you want to discuss Hacktoberfest or anything else regarding auto-cpufreq or its development,[ auto-cpufreq has its Discord server!](https://discord.gg/jYvWNr68)

**Reply by rootCircle** [reply]
Hey! That's great hopping in! :tada:

**Reply by rootCircle** [reply]
Better to also put them on README.md, maybe? :-)

**Reply by AdnanHodzic** [reply]
Also done: https://github.com/AdnanHodzic/auto-cpufreq/#looking-for-developers-and-co-maintainers :)
